.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "✨ Feature request"
+about: Suggest a new feature or idea
+title: ''
+labels: "feature request, awaiting triage"
+assignees: ''
+
+---
+
+<!--
+
+  Before raising this feature request, please make sure you have checked that it isn’t already captured in:
+  - the community backlog: https://design-system.service.gov.uk/community/backlog/
+  - an existing Github issue: https://github.com/alphagov/govuk-frontend/issues
+
+  If you are proposing a new component or pattern, please follow the instructions here: https://design-system.service.gov.uk/community/propose-a-component-or-pattern/
+
+  Please fill in as much of the template below as you’re able to. If you're not sure whether the issue already exists or you're not sure how to fill in the issue template, feel free to open an issue anyway and our team will help you to complete the rest.
+
+-->
+
+## Related component
+<!-- Does this feature relate to an existing component? -->
+
+## Context
+<!-- What are you trying to do? Is this something you think should behave differently, or something that you currently cannot do? Is this related to an existing issue/bug? -->
+
+## Alternatives
+<!-- Can you achieve the same result with an alternative solution? -->
```

Comment:
What are we trying to ask for here? Are we asking for the user to try and come up with other ways we could solve the same problem? Are we trying to understand if the user has a workaround they can make do with?

Reply:
I think this was about understanding if the user has a workaround that they're using to 1) understand what a different solution might look like, and 2) to help us prioritise feature suggestions. But I can see it's not very clear from that description. I've changed it to this to hopefully make it clear that we don't expect a user to try and come up with other solutions if they're not already using one:

```
## Alternatives
<!-- Are you currently using a workaround / alternative solution instead? -->
```